docs/admin/system-information.rst

```diff
@@ -2402,7 +2402,7 @@ subscriptions created in the cluster.
 | ``subowner``        | ``oid`` of the owner of the subscription.| ``INTEGER`` |
 +---------------------+------------------------------------------+-------------+
 | ``subenabled``      | Whether the subscription is enabled,     | ``BOOLEAN`` |
-|                     | always ``true``.                         |             |
+|                     | currently always ``true``.               |             |
```

Comment:
not sure about this - first impression is usually well memorized by a reader -  probably need to hint in a some way, that this table will contain noop value at some point  - here this commitment is probably justified?

Reply:
Usually we try to avoid making any implicit statements about possible future changes. I'd leave it out.